src/components/VideoDetails/VideoDetails.styles.js

```diff
@@ -67,9 +67,8 @@ export const Title = styled.div`
 `;
 
 export const FavoritesButton = styled.button`
-  border-color: #f44336;
-  color: red;
-  background: white;
+  color: ${({ isThemeLight }) => (isThemeLight ? 'red' : 'white')};
```

Comment:
This arrow fn could be extracted into a  variable and then use it in the other different parts for avoiding repetitions.  Something like:

```js
const redOrWhite = ({ isThemeLight }) => (isThemeLight ? 'red' : 'white');
```
```suggestion
  color: ${redOrWhite};
```